### ISSUE TITLE
Accessibility of ui-select

### ISSUE THREAD

**Issue by aeisenberg** [angular-ui/ui-select#527]
We are starting to use ui-select in one of our products. However, we have a requirement that our product be accessible [WCAG 2.0 AA](http://www.w3.org/TR/WCAG20/). In order to meet these requirements, all inputs elements must be titled (eg- have a title attribute, an associated label, etc).  As far as I can tell, ui-select does not meet these requirements.

Am I missing something? Are there any plans to fix this? Would you accept a PR to fix this?

**Comment by amcdnl** [comment]
Thanks for the PR.  I'll review and get back to you.

**Comment by aeisenberg** [comment]
Great. To be clear, this PR is based on work started by @0xF013.

**Comment by user378230** [comment]
Closing as PR was merged.
